blues/slack.py

```diff
@@ -40,12 +41,12 @@ def notify(msg):
     username = blueprint.get('username', 'deploybot')
     icon_emoji = blueprint.get('icon_emoji', ':rocket:')
 
-    endpoint = blueprint.get('endpoint')
-    if not endpoint:
-        warn('No slack API endpoint found, skipping notification')
-        return False
+    endpoint = blueprint.get('endpoint', None)
 
     for channel in channels:
+        if '/' in channel:
+            endpoint, channel = channel.rsplit('/', 1)
```

Comment:
It overwrites default endpoint so if someone writes channel in different order then it will use it for both